frontend/javascripts/oxalis/view/layouting/tracing_layout_view.js

```diff
@@ -159,6 +162,8 @@ class TracingLayoutView extends React.PureComponent<PropsWithRouter, State> {
     storeLayoutConfig(this.currentLayoutConfig, layoutKey, this.currentLayoutName);
   };
 
+  getTabTitle = () => `${this.props.displayName} ${this.props.organization} webknossos`;
```

Comment:
```suggestion
  getTabTitle = () => `${this.props.displayName} ${this.props.organization} webKnossos`;
```

Reply:
Also, I think that some sort of separator between these different parts would be nice. For example an  `|`. If some of these parts happens to be empty or null, unnecessary `|` should be avoided then (for example, filtering the parts as an array and then use `join`).